docker-compose.background.yml

```diff
@@ -28,3 +28,10 @@ services:
     volumes:
       - ./data/redis:/data
     command: redis-server --appendonly yes
+  # hl7_validator_service:
```

Comment:
Added an entry plus comments linking the two since both are necessary. Note I set the timeout to 600s both in the nginx conf and the actual HTTP call, I know that's way too long but until we get session-startup-magic working it needs to be fairly high.